docs/man/xl-pci-configuration.5.pod

```diff
@@ -215,4 +215,18 @@ required to look up the B<BDF> in the list of assignable devices.
 
 =back
 
+=item B<domain>=I<STRING>
+
+=over 4
+
+=item Description
+
+This is domain name with pcid daemon running
+
+=item Default Value
+
+None. Domain name with pcid daemon selects automatically
```

Comment:
It is unclear. What will happen if `domain` will be omitted?

In my understanding, we need to use "Domain-0" by default.

Reply:
No. We need use automatic discovery by default

Reply:
Okay then. I believe we need to squash this change into commit that introduced pcid.
And then you can send the series to xen mailing list, so we can discuss this and other questions with the community